LLHInterface.h

```diff
@@ -70,12 +112,19 @@ class ReceiveLLHOverBifrost : public TemplateLLHGetter
     for(const std::vector<double>& s: fSysts) fBF << s;
     double llh;
     fBF >> llh;
+
+    fSendAsimovOsc = false;
+    fAsimovSysts.clear();
```

Comment:
I know I suggested this clearing behaviour, but it has the potential to confuse the user, if they expect to be able to set the systs, get a likelihood, and then make a small change to the systs. Maybe we should just have a `fSendAsimovSysts` too. I would still just send a length of zero for the case where it's false though.